game/resource/English/ability/items/tooltip_greater_crit.txt

```diff
@@ -4,7 +4,6 @@
 "DOTA_Tooltip_Ability_item_greater_crit_2"                              "Daedalus"
 "DOTA_Tooltip_Ability_item_recipe_greater_crit_2"                       "Daedalus Recipe"
 "DOTA_Tooltip_Ability_item_greater_crit_2_Description"                  "<h1>Passive: Critical Strike</h1>Grants each attack a %crit_chance%%% chance to deal %tooltip_crit_damage%%% additional damage."
```

Comment:
Should be `<h1>Passive: Critical Strike</h1>Grants each attack a %crit_chance%%% chance to deal %crit_multiplier%%% damage.`